### PR TITLE
Migrate to cats effect fs2 3

### PR DESCRIPTION
**Pull request by hogiyogi**

### CODE REVIEW

File: core/src/main/scala/dissonance/Discord.scala [hunk 1]
```diff
@@ -3,16 +3,15 @@ package dissonance
 import java.io.IOException
 
 import cats.Applicative
-import cats.effect._
-import cats.effect.concurrent._
+import cats.effect.std.Queue
+import cats.effect.{Deferred, Ref, Temporal, _}
```
Comment (Billzabob) [inline core/src/main/scala/dissonance/Discord.scala:7]: Got the classic IntelliJ "import these but also everything"

**Comment by hogiyogi597** [comment]
I thought we disabled the code cov stuff @Billzabob

**Comment by Billzabob** [comment]
@hogiyogi597 Nope. I'll do that now

**Comment by Swoorup** [comment]
Any change for a new package build?

**Pull request merged**